woocommerce/woocommerce-gutenberg-products-block
# Create @woocommerce/checkout to export components to a global

Fixes #3613.

This PR:
* Creates a `@woocommerce/blocks-checkout` alias that directs to the `/packages/` directory.
  * Everything in that package is available for external scripts via the `wc.wcBlocksCheckout` global.
  * It should be used to make some components and utils available to extensions.
* Moves the `TotalsItem` component to that package. This is the only component that has been moved for now as a proof-of-concept.
* In the branch history there is an example of usage (2f50349692f53c5bb143516365eee7a98d580dfd). I removed it since that's not something we want to ship, but it can be used as a reference for follow-up PRs.

cc @senadir who created the branch and started the work in this area.

### How to test the changes in this Pull Request:

1. Revert the commit that removes the example code: `git revert 808ef814ce88c2cbda8384700ab779c2dcc6f06f`.
2. Go to the Cart and Checkout and verify there is a `My extension` item in the sidebar:
![imatge](https://user-images.githubusercontent.com/3616980/103780618-82a0d800-5035-11eb-8443-1b2ab3d3d6b1.png)
3. In your browser console, print the values of `wc.wcBlocksCheckout` and verify it includes `TotalsItem` and `SlotFillProvider`.
